scripts/build-worker.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e -u -o pipefail
+
+if [[ -n "${DEBUG-}" ]]; then
+  set -x
+fi
+
+source "$(dirname "$0")/pinned-tools.sh"
+
+echo "Building for: ${WORKER_TYPE} ${BUILD_TARGET} ${SCRIPTING_TYPE}"
+
+pushd "$(dirname "$0")/../"
+  if [[ ${WORKER_TYPE} == "AndroidClient" ]]; then
+      scripts/prepare-unity-mobile.sh "$(pwd)/../logs/PrepareUnityMobile.log"
+  fi
+
+  if [[ ${WORKER_TYPE} == "iOSClient" ]]; then
+      if ! isMacOS; then
+          echo "I can't build for iOS!"
+          exit 1
+      fi
+  fi
+
+  LOG_LOCATION="$(pwd)/../logs/${WORKER_TYPE}-${BUILD_TARGET}-${SCRIPTING_TYPE}.log"
+
+  scripts/build.sh "$(pwd)/../workers/unity" ${WORKER_TYPE} ${BUILD_TARGET} ${SCRIPTING_TYPE} "${LOG_LOCATION}"
```

Comment:
So there is an assumption about where the Unity project is. This is fine for now as they are all in the same place - but we may need to make this generic in the future.

Fine for this PR though